srcpkgs/eza/template

```diff
@@ -11,7 +11,7 @@ license="MIT"
 homepage="https://eza.rocks"
 changelog="https://raw.githubusercontent.com/eza-community/eza/main/CHANGELOG.md"
 distfiles="https://github.com/eza-community/eza/archive/refs/tags/v${version}.tar.gz"
-checksum=0fccd2751568715d708106a23b570201298c2347a360afc522d7029b432f5206
+checksum=2200bc1c07511c105961dcb3fe1682cc9067d55280dac990de7623aff86cab1a
 
 post_patch() {
 	cargo update --package git2@0.18.2 --precise 0.17.2
```

Comment:
Still necessary?

Reply:
sadly yeah, until we sort out the libgit2 1.7 bump